src/components/Message/QuotedMessage.tsx

```diff
@@ -47,19 +47,24 @@ export const QuotedMessage = <
   if (!quotedMessageText && !quotedMessageAttachment) return null;
 
   return (
-    <div className={`${isMyMessage() ? 'quoted-message mine' : 'quoted-message'}`}>
-      {quoted_message.user && (
-        <Avatar
-          image={quoted_message.user.image}
-          name={quoted_message.user.name || quoted_message.user.id}
-          size={20}
-          user={quoted_message.user}
-        />
-      )}
-      <div className='quoted-message-inner'>
-        {quotedMessageAttachment && <Attachment attachments={[quotedMessageAttachment]} />}
-        <div>{quotedMessageText}</div>
+    <div>
```

Comment:
this all looks good, let's just make this a `<>` and `</>` at the end since no code on the div and call it good